lib/KBaseReport/KBaseReportImpl.pm.bak-2017-03-06-15-59-41

```diff
@@ -0,0 +1,1078 @@
+package KBaseReport::KBaseReportImpl;
```

Comment:
yeah, let me clean up those files- there are several bak files in there from before too.